### ISSUE TITLE
Spiral path finishing operation

### ISSUE THREAD

**Issue by Harvie** [GridSpace/grid-apps#17]
Spiral path could be another usefull finishing operation in adition to waterline and linear. Do you think it will be possible to add?

Examples:

![image](https://user-images.githubusercontent.com/276504/36624219-b7f9af80-190c-11e8-8606-f621b2368ada.png)
![image](https://user-images.githubusercontent.com/276504/36624136-eb2c0b56-190b-11e8-8a3f-b84917ef3cbb.png)
![image](https://user-images.githubusercontent.com/276504/36624156-2582b5fc-190c-11e8-858f-60bc5cc40e7d.png)
![image](https://user-images.githubusercontent.com/276504/36624167-3ed2071a-190c-11e8-81e1-5e03726afb37.png)

**Comment by stewartoallen** [comment]
yes, this is possible.  i'll add it to the list.  but realistically won't have time soon.

**Comment by stewartoallen** [comment]
will look into it
